### DISCUSSION TITLE
Is  FixCursorHold.nvim still needed?

### DISCUSSION THREAD

**Discussion by 616b2f** [nvim-neotest/neotest#129] [General]
Hi just was wondering if [FixCursorHold.nvim](https://github.com/antoinemadec/FixCursorHold.nvim) is still needed because on they readme they reference an PR which implements that behavior in neovim 8.0 if I understood it correctly.

**Comment by rcarriga** [comment]
I hadn't actually noticed that message. AFAIK it's still necessary for decoupling from `updatetime` (I've asked here https://github.com/antoinemadec/FixCursorHold.nvim/issues/13) but the README can be reworded to clarify why it's still recommended as the performance issue is fixed
